source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,135 @@
+
+OVA is a known format that is used by different organizations and virtualization platforms. Its specification can be found [here](http://www.dmtf.org/standards/ovf). The standard provides an ["open, secure, portable, efficient and extensible format for the packaging and distribution of software to be run in virtual machines"](https://en.wikipedia.org/wiki/Open_Virtualization_Format).  
+
+Recently, oVirt was enhanced with support for importing OVAs that were generated by VMware. oVirt also makes extensive use of the OVF standard to describe VM configurations, such as VM snapshots and exported VMs.  
+
+The ability to package all the VM data, optionally compressed, in one file eases the processes of backing up a VM and porting a VM to a different platform or hypervisor. Unfortunately, the portability is limited by compatibility issues. Different disk formats, drivers, guest-agents, and OVF structure, hinder the ability to import any OVA to any platform. Therefore, different organizations develop their own conversions tools, like Red Hat's [virt-v2v](http://libguestfs.org/virt-v2v.1.html), for converting VMs from other platforms to their platform.   
+
+# Scope
+This feature is about enhancing the import VM and export VM processes in oVirt by making a broader use of the OVA format.  
+
+## Upload OVA
+Today, the process of importing an OVA is limited to OVA files compatible with those produced by VMware and requires one to make the OVA file accessible by  one of the hosts managed by oVirt with the right file permissions (vdsm:kvm). It is a cumbersome and error-prone process.  
+
+This feature provides a broader and more user-friendly alternative that enables one to upload a local OVA file straight from a client (browser/rest-api based, much like the upload-image functionality that was recently added to oVirt).  
+
+It would be possible to upload OVA files that were generated either by oVirt or by external sources. Officially, the only external source being supported is VMware. However, in practice, the process could work with OVA files generated by other source as long as their OVF configuration contains the data oVirt consumes (in the expected form) and virt-v2v knows how to convert them.
+
+## Export as OVA
+Today, users are provided with two ways to migrate VMs and templates between environments. First, one can use an export domain. However, this approach is suitable only for migration between different oVirt environments. Moreover, there is a plan to drop the support for export domains. Second, one can detach a data domain and attach it to another setup. This way is tailored to disaster recovery and less fits the migration flow due to the side effects in detaching a data domain.  
```

Comment:
Suggested edits (To make my suggestions clearer, I have broken the paragraph into sentences. But it should be written as one paragraph)
Currently, users are provided with two ways of migrating VMs and templates between environments. 

One option is to use an export domain.

However, this approach is only suitable when you wish to migrate between different oVirt environments. 

[comment – what do you mean by ‘different’?]. 

Moreover, support for export domains will be dropped in the future
[can we be more specific – when and who?].

A second option is to detach a data domain and attach it to another setup. This option was designed with disaster recovery in mind. However, because of the side effects associated with detaching a data domain, it is less suited to this migration flow.